Restaurant.js

```diff
@@ -26,12 +26,12 @@ class Restaurant extends React.Component {
     }
 
     return (
-      <li className="media">
+      <li className="media" onClick={this.props.onClick}>
         <div className="media-left">
           <img className="media-object" src={foodTypeImage} />
         </div>
 
-        <div className="media-body">
+        <div className={this.props.selected ? "media-body selected" : "media-body"}>
```

Comment:
There's a way to compose it without writing the same class name twice. 

{"klass" + (condition ? "Foo" : "bar")}